src/static/codeql/scripts/codeql.py

```diff
@@ -364,6 +368,31 @@ def run_codeql_scan(clean=True, scan_builtin_queries=False):
         return [Status.CODEQL_ANALYSIS_FAILED, 0]
     return [Status.SUCCESS, get_severity_issues([Severity.ERROR, Severity.WARNING])]
 
+def set_scan_path(path_arg):
+    r""" Parses the Open Enclave SDK directory path given by the user.
+    :param path_arg:(string) Path to Open Enclave SDK directory
+    :return:(void)
+    """
+
+    if len(path_arg) == 0:
+        return
+
+    global USER_SCAN_DIR
+
+    scan_path = os.path.expanduser(path_arg)
+
+    # If both strings are equal, '~' is not given in the path; path given may be relative
+    if scan_path == path_arg:
+        # If the given path is not an absolute path, generate absolute path
+        if not scan_path.startswith("/"):
+            scan_path = os.path.abspath(scan_path)
+
+    scan_path = os.path.normpath(scan_path)
+
+    if os.path.exists(scan_path):
+        USER_SCAN_DIR = scan_path
+    else:
+        print("Given scan_path ({}) does not exist, using default path".format(scan_path))
```

Comment:
Lets use logger.info instead of print.

Reply:
f"...{scan_path}..." would be good.

Reply:
I kept the verbosity for easier debug IF the `scan_path` argument is used. Less verbose logging is added in lines `429` and `431`